.node-version

```diff
@@ -1 +1 @@
-16.17.1
+18.17.1
```

Comment:
I actually think we'd be better off NOT pinning a specific minor / patch version of node here, so I'm gonna update this to just be `18`